### ISSUE TITLE
[1.20.1] Draconic Evolution compatibility for energy transfer

### ISSUE THREAD

**Issue by coaber** [mekanism/Mekanism-Feature-Requests#559]
### Describe the feature you'd like

Since DE has been updated to 1.18 earlier this year, it allowed for virtually unlimited energy storage, however with that comes the challenge of actually being able to transfer energy at more than the java 32 bit signed limit, of 2.14B, which is included into the base Forge capability system, according to one of the DE devs, it would only be suited for Mek to add Operational Potential ((OP) which is DE's custom energy system that supports longs) in the pool of accepted energy types that mek supports. Again, it already works, but since it does not recognize the energy type, it just falls back to forge's FE type, thus limiting it to 2.14B. Currently, DE is only for 1.18, but they are targeting 1.20, which i assume Mek also does, so perhaps this could be a planned change for when the next major update drops. 

### Describe alternatives you've considered

I've tried different methods such as the quantum entangloporter, however it is also limited, tesseracts, which kind of work if you use multiple in a array, and allow only one for output which results multiple packets sent in one tick, this kind of also works but causes pretty massive hits in server performance. 

### Additional context

none as of yet

**Comment by pupnewfster** [comment]
Will try to look into this in 1.20 then (though I might forget at which point feel free to bump this)

**Comment by coaber** [comment]
Will do.

**Comment by coaber** [comment]
Bumping

**Comment by pupnewfster** [comment]
Draconic Evolution isn't on 1.20 yet?

**Comment by coaber** [comment]
Yeah, it's in the works apparently, no telling when it may see an alpha release. I will comment here once it does.

**Comment by coaber** [comment]
Would this be feasible at this point, considering DE is now out? (Assuming 1.20.1 is still LTS)
